PostHog/PostHogStorage.swift

```diff
@@ -45,7 +55,7 @@ class PostHogStorage {
     init(_ config: PostHogConfig) {
         self.config = config
 
-        appFolderUrl = applicationSupportDirectoryURL() // .appendingPathComponent(config.apiKey)
+        appFolderUrl = getAppFolderUrl(from: config)
```

Comment:
I think we could add a unit test for the final `appFolderUrl` if the `appGroupIdentifier` is set or not